### PR TITLE
[To rel/0.12] make version change easy

### PR DESCRIPTION
**Pull request by mychaow**
## Description

the pr solve two problems:

1. make default value of config file consistent with config in code.
2. make version change easy, only need to change the iotdb-parent version when we want to change project version. Some explain is in https://segmentfault.com/a/1190000019280804.

### Content3 ...

<hr>

This PR has:
- [ ] been self-reviewed.
    - [ ] concurrent read
    - [ ] concurrent write
    - [ ] concurrent read and write 
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. 
- [ ] added or updated version, __license__, or notice information
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious 
  for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold 
  for code coverage.
- [ ] added integration tests.
- [ ] been tested in a test IoTDB cluster.

<hr>

### CODE REVIEW

File: server/src/assembly/resources/conf/iotdb-engine.properties [hunk 1]
```diff
@@ -43,7 +43,7 @@ rpc_port=6667
 # rpc_max_concurrent_client_num=65535
 
 # thrift max frame size, 64MB by default
```
Comment (wangchao316) [inline server/src/assembly/resources/conf/iotdb-engine.properties:45]: hi chao, comments need alter.

File: service-rpc/src/main/java/org/apache/iotdb/rpc/RpcUtils.java [hunk 2]
```diff
@@ -38,7 +38,7 @@
 public class RpcUtils {
 
   /** How big should the default read and write buffers be? Defaults to 64KB */
```
Comment (wangchao316) [inline service-rpc/src/main/java/org/apache/iotdb/rpc/RpcUtils.java:40]: the same...

File: server/src/assembly/resources/conf/iotdb-engine.properties (hunk 1 above)
Comment (mychaow) [reply server/src/assembly/resources/conf/iotdb-engine.properties]: fixed.

File: cluster/src/main/java/org/apache/iotdb/cluster/config/ClusterConfig.java [hunk 3]
```diff
@@ -60,10 +60,10 @@
   private boolean useBatchInLogCatchUp = true;
```
Comment (neuyilan) [inline cluster/src/main/java/org/apache/iotdb/cluster/config/ClusterConfig.java:60]: Hi, the `catchUpTimeoutMS` is not consistent with the iotdb-cluster.properites, in the ClusterConfig.java, the value is 60 seconds, while in the iotdb-cluster.properites, the value is 600 seconds, may be we can change to 300 seconds as the snapshot may take a long time, how do you think?

File: cluster/src/assembly/resources/conf/iotdb-cluster.properties [hunk 4]
```diff
@@ -86,7 +86,7 @@ default_replica_num=3
 # so the leader will send logs(snapshot) to the follower,
 # NOTICE, it may cost minutes of time to send a snapshot,
 # so this parameter should be larger than the snapshot cost time.
-# catch_up_timeout_ms=300000
+# catch_up_timeout_ms=600000
```
Comment (neuyilan) [inline cluster/src/assembly/resources/conf/iotdb-cluster.properties:89]: Hi, the catchUpTimeoutMS is not consistent with the iotdb-cluster.properites, in the ClusterConfig.java, the value is 60 seconds, while in the iotdb-cluster.properites, the value is 600 seconds, may be we can change to 300 seconds as the snapshot may take a long time, how do you think?

Comment (mychaow) [reply cluster/src/assembly/resources/conf/iotdb-cluster.properties:89]: ok

**Review by wangchao316 — APPROVED** [review]
LGTM

**Review by neuyilan — APPROVED** [review]
LGTM

**Comment by jixuan1989** [comment]
about the version management: I can not evaluate the impaction... 
When releasing a new version, we use: 
`mvn release:prepare -DautoVersionSubmodules=true -P site -P compile-cpp`  
and 
`mvn release:perform  -P site -P compile-cpp`

How the command will be after using this PR?

**Comment by jixuan1989** [comment]
current `mvn release:prepare` will change the version of sub modules from ${relativeversion} to a specific version...
So, unless we change the plugins in `release:prepare`, the solution of ${relativeversion} does not take effect..

**Comment by mychaow** [comment]
> current `mvn release:prepare` will change the version of sub modules from ${relativeversion} to a specific version...
> So, unless we change the plugins in `release:prepare`, the solution of ${relativeversion} does not take effect..

ok, I will revert the change.

**Pull request merged**